### PR TITLE
scribus: bump to 1.5.7

### PR DESCRIPTION
**Pull request by kenmays**
Built and tested on Haiku hrev55079.

### CODE REVIEW

**Comment by Begasus** [comment]
Maybe remove the previous version so we can see the changes?

File: app-office/scribus/patches/scribus-1.5.7.patchset [hunk 1]
```diff
@@ -17,21 +17,8 @@ index a8401b8..40860ff 100644
...
-+++ b/cmake/modules/CAIROConfig.cmake
-@@ -19,7 +19,7 @@ else(PREFIX_CAIRO)
-   # use pkgconfig to get the directories and then use these values
-   # in the find_path() and find_library() calls
-   include(FindPkgConfig)
--  pkg_search_module(CAIRO REQUIRED libcairo>=1.14.0 cairo>=1.14.0)
-+#  pkg_search_module(CAIRO REQUIRED libcairo>=1.14.0 cairo>=1.14.0)
- endif(PREFIX_CAIRO)
- 
- find_path(CAIRO_INCLUDE_DIR
```
Comment (Begasus) [inline app-office/scribus/patches/scribus-1.5.7.patchset:32]: This is fixed?

File: app-office/scribus/scribus-1.5.7.recipe [hunk 2]
```diff
@@ -43,7 +43,7 @@ REQUIRES="
 	lib:libpng16$secondaryArchSuffix
 	lib:libpodofo$secondaryArchSuffix
 	lib:libpoppler$secondaryArchSuffix
-	lib:libpython3.7m$secondaryArchSuffix
+	lib:libpython3.9$secondaryArchSuffix
```
Comment (Begasus) [inline app-office/scribus/scribus-1.5.7.recipe:46]: Any specific reason to use python39 here over python3? (not sure how stable python39 is atm)

File: app-office/scribus/patches/scribus-1.5.7.patchset (hunk 1 above)
Comment (kenmays) [reply app-office/scribus/patches/scribus-1.5.7.patchset:32]: Works fine without it.

File: app-office/scribus/scribus-1.5.7.recipe (hunk 2 above)
Comment (kenmays) [reply app-office/scribus/scribus-1.5.7.recipe:46]: Seems OK during build and usage today.

**Comment by Begasus** [comment]
Conflicting files it seems, did you update your master branch before creating this branch recently?

**Pull request merged**